### PR TITLE
[16.0] sale_channel_search_engine: improve ux

### PR DESCRIPTION
**Pull request by hparfr**
add possiblity to resynchronize every binding (products, categories, etc.) add smart button to bindings

### CODE REVIEW

File: sale_channel_search_engine/models/sale_channel.py [hunk 1]
```diff
@@ -9,3 +9,18 @@ class SaleChannel(models.Model):
...
+        action["domain"] = [("index_id", "in", indexes)]
+        action["context"] = "{'hide_res_model': True}"
+        return action
+
+    def synchronize_channel_index_action(self):
+        self._synchronize_channel_index()
+        return self.open_se_binding()
+
+    def _synchronize_channel_index(self):
+        pass
```
Comment (sebastienbeau) [inline sale_channel_search_engine/models/sale_channel.py]: Not sure that we need this (for the product sync is immediatly, and for the category it's depend on the implementation). To discuss

File: sale_channel_search_engine/models/sale_channel.py [hunk 2]
```diff
@@ -9,3 +9,18 @@ class SaleChannel(models.Model):
     _inherit = "sale.channel"
 
     search_engine_id = fields.Many2one("se.backend", "Search Engine")
+
+    def open_se_binding(self):
+        action = self.env.ref("connector_search_engine.se_binding_action").read()[0]
+        indexes = self.search_engine_id.index_ids.ids
+
+        action["domain"] = [("index_id", "in", indexes)]
+        action["context"] = "{'hide_res_model': True}"
```
Comment (hparfr) [inline sale_channel_search_engine/models/sale_channel.py]:
```suggestion

```

Comment (hparfr) [reply sale_channel_search_engine/models/sale_channel.py]: todo me: move functions synhronize in another pr

Comment (hparfr) [reply sale_channel_search_engine/models/sale_channel.py]: moved it : https://github.com/akretion/sale-channel/pull/4

**Comment by hparfr** [comment]
@sebastienbeau changed as you requested

**Pull request merged**